src/Paystack.php

```diff
@@ -278,4 +278,36 @@ private function getData()
         return $this->getResponse()['data'];
     }
 
-}
\ No newline at end of file
+    //Edits by Funsho http://devfunsho.com | @iamfusnho - 22/05/2016
+
+    /**
+     * Create a plan
+     * @return array
+     */
+    public function createPlan(){
+
+        $data = [
+
+            "name" => request()->name,
+            "description" => request()->desc,
+            "amount" => intval(request()->amount),
+            "interval" => request()->interval,
+            "send_invoices" => request()->send_invoices,
+            "send_sms" => request()->send_sms,
+            "currency" => request()->currency,
+        ];
+
+        $this->setRequestOptions();
+
+        $this->response = $this->client->post($this->baseUrl .'/plan', [
+            'body' => json_encode($data)
+        ]);
+
+        return $this->response->getResponse();
```

Comment:
When you have a create method, it shouldn't actually return anything.It should only create. If the user needs to see the details of the new plan created, he can chain it like so:

```php
$paystack->createPlan()->getResponse()
```

Please eliminate the return